### ISSUE TITLE
Docker version not working

### ISSUE THREAD

**Issue by benny1611** [Glimesh/glimesh.tv#448]
Hello, I've tried to run this application in docker (via the docker-compose command you have provided). No matter how I try and how I change the command, I can't get it running. I always get ```standard_init_linux.go:219: exec user process caused: no such file or directory``` in the glimeshtv_app_1 container's logs. What am I doing wrong? And also on what branch can I find a working version?

**Comment by clone1018** [comment]
Apparently our Windows users reported that this is caused by the `run.sh` file having Windows line endings. 

You should be able to run:
``` 
dos2unix run.sh
```

Let me know if that works and I can try and change the repo version.

**Comment by benny1611** [comment]
Ok, that solved the initial problem, but now I've got another problem: 

```
** (UndefinedFunctionError) function Glimesh.Streams.list_categories/0 is undefined or private
(glimesh 0.1.0) Glimesh.Streams.list_categories()
priv/repo/seeds/streams.exs:3: (file)
(elixir 1.10.4) lib/code.ex:926: Code.require_file/2
priv/repo/seeds.exs:3: anonymous fn/2 in :elixir_compiler_2.__FILE__/1
(elixir 1.10.4) lib/enum.ex:2111: Enum."-reduce/3-lists^foldl/2-0-"/3
priv/repo/seeds.exs:2: (file)
(elixir 1.10.4) lib/code.ex:926: Code.require_file/2
(elixir 1.10.4) lib/code.ex:926: Code.require_file/2
[os_mon] memory supervisor port (memsup): Erlang has closed
[os_mon] cpu supervisor port (cpu_sup): Erlang has closed
```

Just so you know, I'm on the dev branch as I have no idea what the main branch is...

**Comment by clone1018** [comment]
`dev` is our main brain, so you are in the right place! I just pushed [a new commit](https://github.com/Glimesh/glimesh.tv/pull/449) to dev to fix that error, can you try again?

**Comment by benny1611** [comment]
Ok, that solved that, now next problem XD:

```
[os_mon] memory supervisor port (memsup): Erlang has closed
[os_mon] cpu supervisor port (cpu_sup): Erlang has closed
npm ERR! Error while executing:
npm ERR! /usr/bin/git ls-remote -h -t ssh://git@github.com/meetecho/janus-gateway.git
npm ERR!
npm ERR! Host key verification failed.
npm ERR! fatal: Could not read from remote repository.
npm ERR!
npm ERR! Please make sure you have the correct access rights
npm ERR! and the repository exists.
npm ERR!
npm ERR! exited with error code: 128
npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-02-12T16_41_10_537Z-debug.log
```

**Comment by benny1611** [comment]
I've tried it a couple of times again, but I get the same error message (deleted the image and rebuild and everything)

**Comment by clone1018** [comment]
Hrm this line should take care of that: https://github.com/Glimesh/glimesh.tv/blob/dev/Dockerfile#L22

But it looks like it's not in our development docker file. It looks like we need to do some maintenance on that in order to get it working again.

**Comment by Eein** [comment]
@clone1018 
fixed by #632

**Comment by clone1018** [comment]
Docker is used now to run the app in production, so this should be resolved. Please let me know it isn't by creating a new issue.
